src/examples/vds_example.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+# Setup the name space:
+import requests
+
+# Start by making a request to the root URI of the VDS API:
+r0 = requests.get("http://odin.rss.chalmers.se/rest_api/v4/vds/")
+
+# The request contains the returned JSON object, which in Python is a
+# dictionary.  This can be printed or inspected to find out its keys and
+# contents.  Let's assume that we have done that, or that we have read
+# the API documentation, so that we know that 'FreqMode' is a key.
+# Use this to single out the frequency mode of interest, in this case 2:
+FM2 = [
+    x for x in r0.json()['VDS']
+    if x['FreqMode'] == 2
+][0]
+
+# Make a new request using the URI provided in the JSON object, and single
+# out the species O3 and the instrument MLS:
+r1 = requests.get(FM2["URL-collocation"])
+O3MLS = [
+    x for x in r1.json()['VDS']
+    if x['Species'] == 'O3' and x['Instrument'] == 'mls'
+][0]
+
+# Repeat for the new object and a date of interest:
```

Comment:
isn't really explained why this is a two step proceedure

Reply:
That is only for clarity.

Reply:
my point is that it's unclear to me what happens and why two steps are needed.

Reply:
Ah, you'd have to read the documentation for a detailed answer to that, but I'll add something to acknowledge it.